lambdas/service/app.py

```diff
@@ -118,12 +118,13 @@
              - [bundles](#operations-Repository-get_repository_bundles)
 
             Azul provides the ability to download metadata in tabular form via
-            the [Manifests](#/Manifests) endpoints. The resulting manifests
-            include links to associated data files and can be used by the
-            [DCP CLI](https://github.com/HumanCellAtlas/dcp-cli) to download the
-            listed files. Manifests can be generated for a selection of files
-            using filters. These filters are interchangeable with the filters
-            used by the [Repository](#/Repository) endpoints.
+            the [Manifests](#operations-tag-Manifests) endpoints. The resulting
+            manifests include links to associated data files and can be used by
+            the [DCP CLI](https://github.com/HumanCellAtlas/dcp-cli) to download
+            the listed files. Manifests can be generated for a selection of
+            files using filters. These filters are interchangeable with the
+            filters used by the [Repository](#operations-tag-Repository)
```

Comment:
```suggestion
            filters used by the [Index](#operations-tag-Index)
```
Based on in progress ticket https://github.com/DataBiosphere/azul/issues/1724, though I could see leaving it `Repository` for now and updating once that lands.

Reply:
Since #1647 will likely land before #1724, I think it makes sense to leave it at Repository.

Reply:
Yes, a PR should be internally consistent and not anticipate changes in other PRs. We use chaining for that.